src/playground/components/Configuration.js

```diff
@@ -109,6 +109,31 @@ export default function Configuration({ rulesMeta, eslintVersion, onUpdate, opti
     const ruleInputRef = useRef(null);
     const [rulesWithInvalidConfigs, setRulesWithInvalidConfigs] = useState(new Set([]));
 
+    const handleRuleChange = () => {
+        selectedRules.forEach(selectedRule => {
+            if (ruleNames.includes(selectedRule)) {
+                options.rules[selectedRule] = ["error"];
+                onUpdate(Object.assign({}, options));
+                ruleInputRef.current.setValue("");
+            }
+        });
+    };
+
+    const Input = props => {
```

Comment:
whenever ```<Configuration/>``` state changes, this ```<Input/>``` component is reinitiated ( since ```<Input/>``` is defined inside ```<Configuration/>```), so every time we choose a rule, we lose focus on the input. So need to handle this as well.

Reply:
✅  will do

Reply:
we can keep the `Input` component definition outside the `Configuration` to fix the focus issue

Reply:
I don't think its happening because of this. I tried moving it out of the Configuration component. Seems like the input field is blurred once the option is selected when we use custom input. 😞

Reply:
Ah, My bad. Even I tried the same just now it didn't solve the issue. It seems like we need to find a different approach to fix it. 😕